tests/fixtures/site.py

```diff
@@ -1,41 +1,150 @@
+import math
+from datetime import datetime, timedelta
+
 import pytest
 
+from django.utils import timezone
 
-@pytest.fixture
-def site_matrix(factories):
+
+def _add_project_resources(project):
+    return [
+        dict(project=project, total_strings=(i + 5) ** 3)
```

Comment:
All those numbers in this file look like magic numbers to me. Can you add a comment explaining what they correspond to? Like, why `+ 5` and why `** 3`?